flight/Modules/Stabilization/stabilization.c

```diff
@@ -108,6 +108,9 @@ static StabilizationSettingsData settings;
 static TrimAnglesData trimAngles;
 static struct pios_queue *queue;
 float gyro_alpha = 0;
+static float gyro_filter_enabled = 0;
```

Comment:
Because when 0 it should indicate that LPF is disabled, 1 should indicate enabled, and 2 should indicate enabled, but Cutoff has changed, so recalculation of gyro_alpha is needed.
For that uint8_t should also be possible.
But as I want to inform GCS when Gyro LPF is disabled (because of to high cutoff frequency) i write to the UAVObject (what you also commented as critical, and maybe need further discussion), and therefore i need a float pointer. 
Maybe that would work also with casting from a uint8_t, have not tried that yet.